stdlib/public/core/RangeReplaceableCollection.swift.gyb

```diff
@@ -509,6 +509,19 @@ public protocol RangeReplaceableCollection
   ///
   /// - Complexity: O(*n*), where *n* is the length of the collection.
   mutating func removeAll(keepingCapacity keepCapacity: Bool /*= false*/)
+
+  /// Removes all elements from the collection.
```

Comment:
This summary seems to be wrong..

Given that the names in the proposal have been changed, and the intent behind having an implementation is that people can get actual experience with the proposal, I think this PR should be updated before we run the review.